Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -204,6 +207,17 @@ export default function ModelExperimental(options) {
   this._attenuation = pointCloudShading.attenuation;
   this._pointCloudShading = pointCloudShading;
 
+  // If the given clipping planes don't have an owner, make this model its owner.
+  // Otherwise, the clipping planes are passed down from a tileset.
+  const clippingPlanes = options.clippingPlanes;
+  if (defined(clippingPlanes) && clippingPlanes.owner === undefined) {
+    ClippingPlaneCollection.setOwner(clippingPlanes, this, "_clippingPlanes");
+  } else {
+    this._clippingPlanes = clippingPlanes;
+  }
+  this._clippingPlanesState = 0; // Used for checking if shaders need to be regenerated due to clipping plane changes.
```

Comment:
It took me a while to figure out what this state variable is doing, I had to stare at `ClippingPlaneCollection.clippingPlanesState` for a moment to understand it's just trying to have a single value to check if the settings change. 

Could we change this comment to explain that all ModelExperimental cares is that if this value changes, the shader should be updated?

Maybe also update the JSDoc for `ClippingPlaneCollection` to make it clearer that this is the intended use?